examples/deepwalk/model.py

```diff
@@ -28,53 +28,85 @@ def __init__(self,
                  neg_num=5,
                  num_emb_part=1,
                  sparse=False,
-                 sparse_embedding=False):
+                 sparse_embedding=False,
+                 shared_embedding=False):
         super(SkipGramModel, self).__init__()
 
         self.num_nodes = num_nodes
         self.neg_num = neg_num
+        self.shared_embedding = shared_embedding
 
         # embed_init = nn.initializer.Uniform(
         # low=-1. / math.sqrt(embed_size), high=1. / math.sqrt(embed_size))
-        embed_init = nn.initializer.Uniform(low=-1.0, high=1.0)
-        emb_attr = paddle.ParamAttr(
-            name="node_embedding", initializer=embed_init)
+        u_embed_init = nn.initializer.Uniform(low=-1.0, high=1.0)
```

Comment:
Please remove all prefixes  ```u_``` in model.py and delete  ```print(model.state_dict())``` in train.py. Then the code can be merged.